docker/all-in-one/etc/postgresql/logging.conf

```diff
@@ -31,3 +31,5 @@ log_rotation_size = 0		# Automatic rotation of logfiles will
 					# or size-driven rotation.  Default is
 					# off, meaning append to existing files
 					# in all cases.
+
+log_disconnections = on			# log end of a session, including duration
```

Comment:
intentional?

Reply:
Yes. 
This param is disabled by default and can't track disconnections any other way since the connections are terminated and thus not available in `pg_stat_activity`.
This allows the shutdown script to track when the last disconnection happened and shutdown the machine after X minutes.

Postgres output related to disconnections is filtered out by Vector, so it won't additionally be shipped to Logflare.

This approach diverges the configs, thus we could have the shutdown script run an `ALTER SYSTEM` at startup to set `log_disconnections` to on, since it's a shutdown script dependency after all.